superdesk/publish/__init__.py

```diff
@@ -13,14 +13,16 @@
 
 from superdesk.celery_app import celery
 from superdesk.publish.publish_content import PublishContent
+from superdesk import get_backend
+
 
 logger = logging.getLogger(__name__)
 
 transmitters = {}
 transmitter_errors = {}
 
-subscriber_types = ['broadcast', 'digital', 'wire']
-SUBSCRIBER_TYPES = namedtuple('SUBSCRIBER_TYPES', ['BROADCAST', 'DIGITAL', 'WIRE'])(*subscriber_types)
+subscriber_types = ['broadcast', 'digital', 'wire', 'all']
+SUBSCRIBER_TYPES = namedtuple('SUBSCRIBER_TYPES', ['BROADCAST', 'DIGITAL', 'WIRE', 'ALL'])(*subscriber_types)
```

Comment:
The subscriber type needs to be declared in `vocabularies.json` file. Hope that's been taken care off.